Tests/varLib/varLib_test.py

```diff
@@ -813,7 +813,32 @@ def test_designspace_fill_in_location(self):
 
         assert ds_loaded.instances[0].location == {"weight": 0, "width": 50}
 
+    def test_varlib_build_incompatible_features(self):
+        try:
+            self._run_varlib_build_test(
+                designspace_name="IncompatibleFeatures",
+                font_name="IncompatibleFeatures",
+                tables=["GPOS"],
+                expected_ttx_name="IncompatibleFeatures",
+                save_before_dump=True,
+            )
+        except VarLibMergeError as e:
```

Comment:
if you want to test whether a specific exception has been raised you should use `pytest.raises` as context-manager like so:

```python
with pytest.raises(VarLibMergeError, match="error message substring or pattern to be matched"):
    # do someting that triggers the exception
```

it will also correctly handle the wrong exception or no exception cases

reference: https://docs.pytest.org/en/stable/reference.html#pytest-raises

Reply:
Oops, I saw `class BuildTest(unittest.TestCase)` at the top and assumed this was still unittest based.